android/src/main/java/cl/json/social/EmailShare.java

```diff
@@ -26,6 +26,11 @@ protected String getPackage() {
         return PACKAGE;
     }
 
+    @Override
+    protected String getComponentClass() {
+        return null;
```

Comment:
Any time a bunch of objects extend a parent object, and you need one thing in one of the children but none of the others, you should just provide a default implementation in the parent and override it in the one child that needs it

Alternatively, just override the one method you want to alter in the child (getPackage())

Touching all the other classes doesn't make sense to me

Reply:
Agreed

Reply:
Addressed your comments. Please check again.